src/rdkafka_cgrp.c

```diff
@@ -3152,6 +3442,19 @@ rd_kafka_cgrp_op_serve (rd_kafka_t *rk, rd_kafka_q_t *rkq,
                 rko = NULL;
                 break;
 
+        case RD_KAFKA_OP_CG_METADATA:
+                /* Return the current consumer group metadata. */
+                rko->rko_u.cg_metadata = rkcg->rkcg_member_id
+                        ? rd_kafka_consumer_group_metadata_new_with_genid(
+                                rkcg->rkcg_rk->rk_conf.group_id_str,
+                                rkcg->rkcg_generation_id,
+                                RD_KAFKAP_STR_DUP(rkcg->rkcg_member_id),
```

Comment:
The dupped member_id is leaked here, right? (did this pass asan/valgrind?)

Reply:
Suggest `rkcg->rkcg_member_id->str` as rkcg_member_id is safe to use since it is new():ed

Reply:
done